### ISSUE TITLE
[BUILD] Git master build fails

### ISSUE THREAD

**Issue by prokoudine** [sobotka/olive#27]
**Platform**

Ubuntu 19.10, 64-bit

Commit 6ea6dd810e07957fb73510d1cd83dec08c1cd23b

**Additional Information / Output**

```
/usr/include/OpenImageIO/fmath.h:872:1: note: candidate: 'template<class S, class D> D OpenImageIO_v2_0::convert_type(const S&)'
  872 | convert_type (const S &src)
      | ^~~~~~~~~~~~
/usr/include/OpenImageIO/fmath.h:872:1: note:   template argument deduction/substitution failed:
/home/alex/soft/video/olive/olive-2020-03-31/app/render/color.cpp:9:95: note:   candidate expects 1 argument, 5 provided
    9 |                      PixelFormat::FormatHasAlphaChannel(format) ? kRGBAChannels : kRGBChannels);
      |                                                                                               ^
make[2]: *** [app/CMakeFiles/olive-editor.dir/build.make:2330: app/CMakeFiles/olive-editor.dir/render/color.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:148: app/CMakeFiles/olive-editor.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

**Comment by sobotka** [comment]
I'll confirm this. I'm getting a similar result:

```
/olive-master/app/render/color.cpp: In constructor ‘Color::Color(const char*, const PixelFormat::Format&)’:
/olive-master/app/render/color.cpp:9:95: error: no matching function for call to ‘convert_type(OpenImageIO_v2_0::TypeDesc::BASETYPE, const char*&, OpenImageIO_v2_0::TypeDesc::BASETYPE, float [4], const int&)’
     PixelFormat::FormatHasAlphaChannel(format) ? kRGBAChannels : kRGBChannels);
```

**Comment by itsmattkc** [comment]
Fixed in 5e609882d84b100a8b6c9f8285bf1af12a11c0aa
